en/docs/administer/product-security/logins-and-passwords/working-with-encrypted-passwords.md

```diff
@@ -88,31 +96,33 @@ The instructions below explain how plain text passwords in configuration files c
     truststrore_password  = "FEQvzFCYpJfuD9XYb0H4KW/lZICu7vaPMzC02kIIbMy47q1q68QxIdOAwQxv+wiRAI/fQIxX6ygTjWV+3bjYmP1Mj0qrhx6CeH6L467ISIHT8oTdecqq5kcGqMEBanexQ7Wu/ULeiiaGLA4x9OXHgJsKd9x3yf6Vm56FzmVkUM5+HAX6pYpBSEFuDKJprhlJtvrEN//nGb0p342g4CqG9VqW3UFbkQaawmItSd9pMnBlDM6+STmiDBUFrV7gdJfrCzGnJs7QBl20Kxg5aznNkwnpJ3bEwj3Trkzsxujk1wMOnhk5XvwdeKRevkX8MyHrnUICyXZ6TK0po1wrYZjYvw=="
     ```
 
-7.  Please refer [Resolving already encrypted passwords during server startup](#resolving-already-encrypted-passwords-during-server-startup) in order to resolve the passwords during different modes of server startup.
+    To resolve the passwords during different modes of server startup, see [Resolving already encrypted passwords during server startup](#resolving-already-encrypted-passwords-during-server-startup).
 
 ## Encrypting secured endpoint passwords
 
-When exposing an API backend which is secured with [Digest](({{base_path}}/learn/design-api/endpoints/endpoint-security/basic-auth)) or [Basic]({{base_path}}/learn/design-api/endpoints/endpoint-security/digest-auth) Authentication, the backend user credentials has to be provided under endpoint configuration. These credentials are encoded in base64 and stored in API configuration as Basic Authorization header(`Authorization: Basic base64Encode(<username>:password)`). By default, the Authorization header value is stored in plain text.
+When exposing an API backend, which is secured with [Digest]({{base_path}}/learn/design-api/endpoints/endpoint-security/digest-auth) or [Basic]({{base_path}}/learn/design-api/endpoints/endpoint-security/basic-auth) Authentication, the backend user credentials have to be provided under endpoint configuration. These credentials are encoded in base64 and stored in API configuration as Basic Authorization header (`Authorization: Basic base64Encode(<username>:password)`). By default, the Authorization header value is stored in plain text.
```

Comment:
and stored in the API configuration

Reply:
it's already mentioned

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/789/commits/9ba49a0403a2a0d620b949ed2475a8cb46a66b87

Reply:
@bhathiya  was emphasizing on the useage of "the"

Reply:
I see. Btw, you don't use the Github suggestions feature? It makes life easy and reduces confusion. Have a try.